src/page/Registry.js

```diff
@@ -217,7 +217,7 @@ async function renderDir(pathname, entry) {
       const rawUrl =
         entryType === "esm"
           ? `${entry.url}${path}README.md`
-          : `${entry.url}README.md`;
+          : `https://raw.githubusercontent.com/${owner}/${repo}/${entry.branch}/${path}README.md`;
```

Comment:
What about non-github entries?

Reply:
This is already in a `if (entryType === "github" || entryType === "esm")` block (line 176) so it must be `github` because its not `esm`.